teams.md/src/pages/csharp.tsx

```diff
@@ -1,5 +1,7 @@
-import { Redirect } from "@docusaurus/router";
+import { Redirect } from '@docusaurus/router';
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 export default function CSharp() {
-    return <Redirect to='csharp/getting-started' />;
-}
\ No newline at end of file
+  const baseUrl = useBaseUrl('/');
+  return <Redirect to={`${baseUrl}csharp/getting-started`} />;
```

Comment:
This fixes an issue where redirects were getting a second language path segment added in when Ctrl-clicking a link or changing language context.